include/cute/atom/mma_atom.hpp

```diff
@@ -599,6 +599,46 @@ make_tiled_mma(MMA_Op       const&,
   return make_tiled_mma(MMA_Atom<MMA_Op>{}, thr_layout, permutations);
 }
 
+// This helper fn adopts the approach described in
+// media/docs/cute/0t_mma_atom.md#tiledmmas to construct a scatter
+// permutation which ensures all sub-groups operate on contiguous
+// chunks of the MMA. The docs describe how the Layout
+// (i.e. SubgroupLayout) implies a repetition of the atom across additional
+// hardware. Permutations, in the simplest form, imply additional iterations
+// to cover a larger tile (i.e. CTATileShape) than the hardware can handle
+// at once. By specifying additional modes and strides, we can achieve 
+// arbitrary TiledMMA layouts.
+template <typename MMA_Atom, typename CTATileShape, typename SubgroupLayout>
+struct ContigBlockMMAHelper{
+private:
+  using AtomShape = typename MMA_Atom::Shape_MNK;
+
+  // Represents the canonical MMA block which would be handled
+  // by these subgroups without permutation info...
+  static constexpr auto DefaultBlock =
```

Comment:
why is this called "Default". It depends on the user provided SubgroupLayout.

Reply:
How about something like?
```suggestion
  static constexpr auto SGBlockShape =
```

Reply:
The `DefaultBlock`, `DefaultTiler` and `DefaultTiling` represent the canonical TiledMMA layout for the given atom and sub-group layout. In other words, this is the layout in the absence of a permutation. More accurately, it's the layout in the absence of a non-layout-left permutation.

Importantly, `DefaultBlock` *doesn't* represent the block handled by a single sub-group. Effectively, it is the block that would be handled by *all sub-groups working together in a single iteration*. The reason I'm constructing it is that it tells us how many iterations we need to cover the whole CTA tile.

I've changed `s/Default/Canonical/` because I think it better describes it. Does that seem better?

Reply:
I've opted for `CanonicalBlockShape`, because as you say we only use the Shape. However, note that this is *not* the block shape for a single sub-group. This is the block shape that would be obtained by all (e.g. 8x4) sub-groups working together on a block of the CTA tile. This is the default/canonical tiling strategy, so I think this name is appropriate.